CHANGELOG.md

```diff
@@ -6,6 +6,11 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [PEP 440](https://www.python.org/dev/peps/pep-0440/)
 and uses [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [0.9.10]
+
+### Fixed
+- The [`update_asf_tools_version`](.github/workflows/update_asf_tools_version.yml) and [`update_sdk_version`](.github/workflows/update_sdk_version.yml) GitHub Actions workflows now uses the `gh` CLI instead of the archived `repo-sync/pull-request` action.
```

Comment:
```suggestion
- The [`update_asf_tools_version`](.github/workflows/update_asf_tools_version.yml) and [`update_sdk_version`](.github/workflows/update_sdk_version.yml) GitHub Actions workflows now use the `gh` CLI instead of the archived `repo-sync/pull-request` action.
```